lib/views/warning_detail_view.dart

```diff
@@ -465,417 +327,720 @@ class _DetailScreenState extends ConsumerState<DetailScreen> {
+      List<LatLng> polygonPoints =
+          Area.getListWithAllPolygons(alert.info.first.area);
+
+      if (polygonPoints.isNotEmpty) {
+        return CameraFit.bounds(
+          bounds: LatLngBounds.fromPoints(polygonPoints),
+          padding: const EdgeInsets.all(30),
+        );
+      } else {
+        return CameraFit.bounds(
+          // set the bounds to the northpol if we don't have any points
+          bounds: LatLngBounds.fromPoints([const LatLng(90.0, 0.0)]),
+          padding: const EdgeInsets.all(30),
+        );
+      }
+    }
+
+    return SizedBox(
+      height: 200,
+      child: MapWidget(
+        mapController: mapController,
+        initialCameraFit: createInitCameraFit(),
+        polygonLayers: [
+          //@todo can be null
+          PolygonLayer(
+            polygons: Area.createListOfPolygonsForAreas(alert.info.first.area),
+          ),
+        ],
+        widgets: [
+          Align(
+            alignment: Alignment.bottomLeft,
+            child: Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: FloatingActionButton(
+                tooltip: localizations
+                    .warning_detail_view_map_center_map_button_tooltip,
+                onPressed: () {
+                  mapController.fitCamera(createInitCameraFit());
+                },
+                child: const Icon(Icons.center_focus_strong),
+              ),
+            ),
+          ),
+        ],
       ),
     );
   }
 }
+
+class _Description extends StatelessWidget {
+  const _Description({required this.alert});
+
+  final WarnMessage alert;
+
+  @override
+  Widget build(BuildContext context) {
+    var localizations = context.localizations;
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Row(
+          children: [
+            const Icon(Icons.description),
+            const SizedBox(width: 5),
+            Text(
+              localizations.warning_description_title,
+              style: TextStyle(
+                fontWeight: FontWeight.bold,
+                fontSize: userPreferences.warningFontSize + 5,
+              ),
+            ),
+          ],
+        ),
+        const SizedBox(height: 2),
+        SelectableText.rich(
+          TextSpan(
+            children: _htmlTextToTextSpans(alert.info[0].description),
+            style: TextStyle(fontSize: userPreferences.warningFontSize),
+          ),
+        ),
+      ],
+    );
+  }
+}
+
+class _WarningAppendix extends StatelessWidget {
+  const _WarningAppendix({required this.assets});
+
+  final List<Widget> assets;
+
+  @override
+  Widget build(BuildContext context) {
+    var localizations = context.localizations;
+
+    return Column(
+      children: [
+        Row(
+          children: [
+            const Icon(Icons.image),
+            const SizedBox(width: 5),
+            Text(
+              "${localizations.warning_appendix}:",
```

Comment:
We can remove the `:` for the headline and, therefore, the formatted string.
```suggestion
              localizations.warning_appendix,
```